tool/randomizer.lua

```diff
@@ -123,10 +138,8 @@ end
 function randomizer.reset()
     _G.onlevelchange = nil
     _G.onreset = nil
```

Comment:
Would it be a good idea to use `_G.onreset()` instead of `resetfunction()` to create a new sequence on `New Game`?

Reply:
That might make sense now that such a function exists :) Just keep in mind that there can only be a single event handler at the same time.